src/graphql/operations/votes.ts

```diff
@@ -83,7 +91,7 @@ export default async function(parent, args, context?, info?) {
   if (requestedFields.proposal && votes.length > 0) {
     const proposalIds = votes.map(vote => vote.proposal);
     const query = `
-      SELECT p.*, spaces.settings FROM proposals p
+      SELECT p.*, spaces.settings, spaces.created_at as space_created_at, spaces.updated_at as space_updated_at FROM proposals p
```

Comment:
Can you use uppercase for MySQL commands and remove _at if you can like this
```suggestion
      SELECT p.*, spaces.settings, spaces.created_at AS space_created, spaces.updated_at AS space_updated FROM proposals p
```

Reply:
integrated changes in all files